.github/workflows/ci.yml

```diff
@@ -29,23 +29,24 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        config: [ {python-version: "3.8", oldest-pymc: false}, {python-version: "3.11", oldest-pymc: true}]
+        python-version: ["3.9", "3.11"]
```

Comment:
This is gonna double the number of jobs, we were doing oldest pymc and oldest python and most recent pymc and most recent python. I think that's enough

Reply:
oh, good catch, actually I didn't think of that result.